fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -1521,7 +1552,7 @@ createMaskingPolicyStatement
     ;
 
 dropMaskingPolicyStatement
-    : DROP (IF EXISTS)? MASKING POLICY policyName=qualifiedName FORCE?
+    : DROP MASKING POLICY (IF EXISTS)? policyName=qualifiedName FORCE?
```

Comment:
Has the old grammar been released?

Reply:
i think it hasn't been used because it's an unreleased enterprise feature.